README.md

```diff
@@ -105,6 +107,12 @@ helpers.getShopperToken(config, { type: "guest" }).then(async (token) => {
 });
 ```
 
+### Error Handling
+
+SDK methods return an appropriate object by default when the API call returns a successful response. The object is built from the body of the response. If the API response is not successful, an Error is thrown. The error message is set to the status code plus the status text. The Error object includes a custom 'response' attribute with the entire Response object for inspection. To access the body of the response when an error occurs, see sample code above.
```

Comment:
Can we add one concrete example to this to make it clear why someone would want to do this, e.g. in the case where you want to see the server response for why a request is bad (400)?

Reply:
Added an example, let me know what you think